wcivf/apps/elections/templates/elections/includes/_hustings_list.html

```diff
@@ -3,6 +3,6 @@
 {% for husting in hustings %}
 <p>
 {% if husting.in_past %}(Past event) {% endif %}
-<strong>{{ husting.starts|naturalday:"l j F Y"|title }} {{ husting.starts|date:"H:i"}}{% if husting.ends %}&ndash;{{ husting.ends | date:"H:i"}}{% endif %}:</strong> {% if husting.url %}<a href="{{ husting.url }}">{% endif %}{{ husting.title}}{% if husting.url %}</a>{% endif %}, {{ husting.location }} {{ husting.postcode }}. {% if husting.postevent_url %}<a href="{{ husting.postevent_url }}"><i class="icon-videocam" aria-hidden="true"></i> See video or other info from this event.</a>{% endif %}
+<strong>{{ husting.starts|naturalday:"l j F Y"|title }} {{ husting.starts|date:"H:i"}}{% if husting.ends %}&ndash;{{ husting.ends | date:"H:i"}}{% endif %}:</strong> {% if husting.url %}<a href="{{ husting.url }}">{% endif %}{{ husting.title}}{% if husting.url %}</a>{% endif %}{% if husting.location %}, {{ husting.location }} {{ husting.postcode }}{% endif %}{% if husting.postevent_url %}<br><a href="{{ husting.postevent_url }}"><i class="icon-videocam" aria-hidden="true"></i> See video or other info from this event.</a>{% endif %}
```

Comment:
I know this is how it was previously, but can we have some line breaks here, for legibility?